GameOfLifeApp/Communicator.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Requests info from  user. Sends a message about incorrect input.
+    /// </summary>
+    public class Communicator
+    {
+        const string notANumberMessage = "Inputed value is not a number. Please input a number. Press any key to continue.";
+        const string outOfLimit = "Out of limit.";
+        const string notMinMax = "Value should be more than {0} and less than {1}." + "\n" + "Press any key to continue.";
+
+        /// <summary>
+        /// Gets data from user.
+        /// </summary>
+        /// <param name="message">Requests to provide informatiom about array size.</param>
+        /// <returns>Returns count of columns and count of rows.</returns>
+        private static string GetInput(string message)
+        {
+            Console.Clear();
+            Console.WriteLine(message);
+            return Console.ReadLine();
+        }
+
+        /// <summary>
+        /// Checks the input and converts it to int. Requests to input a number.
+        /// </summary>
+        /// <param name="message">User`s response.</param>
+        /// <returns>Returns a number that equals to the count of rows / columns.</returns>
+        public int GetNumericResponse(string message)
+        {
+            do
+            {
+                string? response = GetInput(message);
```

Comment:
there is no need to add ? as string is already nullable by default as it is a reference type

Reply:
Done.